src/test/jtx/impl/Env.cpp

```diff
@@ -537,6 +537,14 @@ Env::do_rpc(std::vector<std::string> const& args)
     return response;
 }
 
+void
+Env::enableFeature(uint256 const feature)
```

Comment:
I guess we can make it a free function and add the `hasFeature` option. Given the way we handle existing transfer rates, there's no real need for this function in _this_ PR, and it can be coordinated with the other PR.